[DOCS] Alerts related by session ID and Alerts related by process ancestry no longer beta and don't require a feature flag (#2527)
elastic/security-docs: docs/detections/alerts-view-details.asciidoc

* *Cases related to the alert* - Shows the total number and names of cases to which the alert has been added. Click a case's name to open its details.
* *Alerts related by source event* - Shows the ten most recent alerts created by the same source event. This can help you find alerts with a shared origin and provide more context about the source event. Click the *Investigate in timeline* button to examine related alerts in Timeline.

If you have a https://www.elastic.co/pricing[Platinum or Enterprise subscription], these details are also included:

> * *Alerts related by session ID* - Shows the ten most recent alerts generated during the same <<session-view, session>>. These alerts share the same session ID, which is a unique ID for tracking a given Linux session. To use this feature, you must enable the *Include session data* setting on your {elastic-defend} integration policy. Refer to <<enable-session-view, Enable Session View data>> for more information.

Reviewer: Prepositions are tricky, but I like to thing of config settings are "within" a window or inside, hence `in`. But it's not crucial.  
```suggestion
* *Alerts related by session ID* - Shows the ten most recent alerts generated during the same <<session-view, session>>. These alerts share the same session ID, which is a unique ID for tracking a given Linux session. To use this feature, you must enable the *Include session data* setting in your {elastic-defend} integration policy. Refer to <<enable-session-view, Enable Session View data>> for more information.
```

Author: I think both work, but "in" sounds a bit more correct when I think about where the setting is located _within_ the policy.

* *Cases related to the alert* - Shows the total number and names of cases to which the alert has been added. Click a case's name to open its details.
* *Alerts related by source event* - Shows the ten most recent alerts created by the same source event. This can help you find alerts with a shared origin and provide more context about the source event. Click the *Investigate in timeline* button to examine related alerts in Timeline.

> * *Alerts related by session ID* - Shows the ten most recent alerts generated during the same <<session-view, session>>. These alerts share the same Session ID, which is a unique ID for tracking a given Linux session. To use this feature, the *Include session data* setting must be enabled on your {elastic-defend} integration policy. Refer to <<enable-session-view, Enable Session View data>> for more information.

Reviewer 2: Revising passive voice. Also, "Session ID" prob doesn't need to be capitalized; I don't think it's a proper name, just the ID for a given session.
```suggestion
* *Alerts related by session ID* - Shows the ten most recent alerts generated during the same <<session-view, session>>. These alerts share the same session ID, which is a unique ID for tracking a given Linux session. To use this feature, you must enable the *Include session data* setting on your {elastic-defend} integration policy. Refer to <<enable-session-view, Enable Session View data>> for more information.
```

+

> NOTE: This feature requires a https://www.elastic.co/pricing[Platinum or Enterprise subscription].

Reviewer 2: Having the same note for multiple items feels repetitive and disrupts the flow. I'm wondering if we could consolidate the info, something like:

> The Insights section provides the following details:
> - Cases related to the alert
> - Alerts related by source event
> 
> If you have a Platinum or Enterprise subscription, these details are also included:
> - Alerts related by session ID
> - Alerts related by process ancestry

Author: Yeah, agree about the notes breaking up the flow. I like this ^
